### PR TITLE
Add revert hashes flag

### PR DESCRIPTION
**Pull request by Mr-Leshiy**

### CODE REVIEW

File: include/veriblock/pop/blockchain/block_index.hpp [hunk 1]
```diff
@@ -352,21 +352,31 @@ struct BlockIndex : public Block::addon_t {
...
         header->toPrettyString(),
         addon_t::toPrettyString());
   }
 
-  std::string toShortPrettyString() const {
-    return format("{}:{}:{}", Block::name(), height, HexStr(getHash()));
+  std::string toShortPrettyString(bool reverse_hash = false) const {
+    auto hash = getHash();
+    if (reverse_hash) {
+      hash = std::vector<uint8_t>(hash.rbegin(), hash.rend());
```
Comment (Warchant) [inline include/veriblock/pop/blockchain/block_index.hpp:376]: can be replaced with 

```
HexStr(hash.rbegin(), hash.rend());
```

**Comment by wahidtanner** [comment]
Why do we sometimes have reversed hashes?

**Comment by Mr-Leshiy** [comment]
Will close this pr, it is better to do it on the vBTC side
https://github.com/VeriBlock/vbk-ri-btc/pull/542

**Comment by Mr-Leshiy** [comment]
@wahidtanner , it is needed for the original vBTC logs and library logs correspondence https://track.veriblock.org/issue/ALT-658

**Pull request closed without merging**